tests/handler/index.test.ts

```diff
@@ -150,4 +239,28 @@ describe('Test lambda handler', () => {
 
     await expect(handler(eventMock, contextMock)).rejects.toThrow('oops!');
   });
+
+  test('should throw error when failed to filter ATFs with no availability', async () => {
```

Comment:
Does this test still pass?

Reply:
Fixed in my latest commit - I now wrap the filtering in a try / catch just in case something does fail - but it will just log the error and continue